controllers/admin/AdminTranslationsController.php

```diff
@@ -3047,8 +3053,11 @@ protected function getAllModuleFiles($modules, $root_dir = null, $lang, $is_defa
      */
     public function initFormModules()
     {
-        // Get list of modules
-        $modules = $this->getListModules();
+        // Get list of installed modules
+        $installed_modules = $this->getListModules();
+
+        // get selected module
+        $modules[0] = Tools::getValue('module');
```

Comment:
There are some functions that have an array for entry, so to not break all this functions I've chosen to have an array that contains one module.